Service/ResponseProcessor.php

```diff
@@ -64,8 +64,8 @@ public function processResponse(CommandInterface $command, ResponseInterface $re
         try {
             /** @var array<string, array<string, mixed>> $data */
             $data = \json_decode($content, true, 512, \JSON_THROW_ON_ERROR);
-        } catch (\Exception) {
-            throw new CentrifugoException('Centrifugo response payload is not a valid JSON');
+        } catch (\Exception $parsingException) {
+            throw new CentrifugoException($response, 'Centrifugo response payload is not a valid JSON', previous: $parsingException);
```

Comment:
If you use name of arguments, like `previous: ` then use names for all arguments.